requirements/requirements-ci.txt

```diff
@@ -74,7 +74,7 @@ imagesize==1.1.0          # via -r requirements-dev.txt, sphinx
 importlib-metadata==2.0.0  # via -r requirements-dev.txt, pluggy
 incremental==17.5.0       # via -r requirements-dev.txt, treq, twisted
 iniconfig==1.0.1          # via -r requirements-dev.txt, pytest
-ipfshttpclient==0.6.0.post1  # via -r requirements-dev.txt, web3
+ipfshttpclient==0.7.0a1   # via -r requirements-dev.txt, web3
```

Comment:
I get
```
ERROR: Could not find a version that satisfies the requirement ipfshttpclient==0.7.0a1 (from -r /tmp/tmpu8lkdzf1 (line 14)) (from versions: 0.4.10, 0.4.11, 0.4.12, 0.4.13, 0.4.13.1, 0.4.13.2, 0.6.0, 0.6.0.post1, 0.6.1)
ERROR: No matching distribution found for ipfshttpclient==0.7.0a1 (from -r /tmp/tmpu8lkdzf1 (line 14))
```
locally, now. Did the release just disappear, or is something wrong with my local state? @fredo

Reply:
works for me locally. Have you tried a new venv?

Reply:
That does indeed help. But I have no idea what kind of local state can provoke that error message.